src/update/UpdateManager.h

```diff
@@ -0,0 +1,57 @@
+/*!********************************************************************
+ Audacity: A Digital Audio Editor
+
+ @file UpdateManager.h
+ @brief Declare a class that managing of updates.
+
+ Anton Gerasimov
+ **********************************************************************/
+#pragma once
+
+#include "VersionId.h"
+#include "VersionPatch.h"
+#include "UpdateDataParser.h"
+
+#include "Project.h"
+#include "Prefs.h"
+
+#include <wx/string.h>
+#include <wx/event.h>
+#include <wx/timer.h>
+
+/// A class that managing of updates.
+/**
+    Opt-in request and show update dialog by the scheduled time.
+    Have a built-in check that allows avoiding multiplying update notifications
+    when multiple Audacity windows are shown.
```

Comment:
Do we still need comments about multiple windows after the other corrections for object lifetimes?